admin_guide/managing_projects.adoc

```diff
@@ -0,0 +1,178 @@
+= Managing Projects
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+In OpenShift, projects are used to isolate content from groups of developers. As an OpenShift administrator, you can give developers access to certain projects, allow them to create their own, and give them administrator rights.
+
+[[selfprovisioning-projects]]
+
+== Self-provisioning Projects
+
+You can allow developers to create their own projects. There is an endpoint
+that will provision a project according to a
+link:../dev_guide/templates.html[template]. The web console and `oc new-project` 
+command use this endpoint when a developer link:../dev_guide/projects.html[creates a new project].
+
+[[modifying-the-template-for-new-projects]]
+=== Modifying the Template for New Projects
+The API server automatically provisions projects based on the template that is
+defined in the `projectRequestTemplate` parameter of the *_master-config.yaml_*
+file. If the parameter is not defined, the API server creates a default template
+that creates a project with the requested name, and assigns the requesting user
+to the "admin" role for that project.
+
+To create your own custom project template:
+
+. Start with the current default project template:
++
+----
+$ oadm create-bootstrap-project-template -o yaml > template.yaml
+----
+
+. Modify the template by adding objects or modifying existing objects, then load the template:
```

Comment:
You modify the template by editing the file. Then you load the file as below. Yes, it's two steps.